tests/api/mines.api.test.ts

```diff
@@ -0,0 +1,33 @@
+import { createApiTestEnvironment } from '../environments'
+import { MongodbSpies, mockClient } from '../../__mocks__/mongodb';
+
+const environment = createApiTestEnvironment();
+environment.initialize();
+
+jest.mock('mongodb');
+
+describe('Mines API Test', () => {
+    const {
+        findSpy,
+    }: MongodbSpies = jest.requireMock('mongodb');
+
+    beforeEach(() => {
+        findSpy.mockClear();
+    });
+
+    it('should return 200', async () => {
+
+        //TODO replace fastify.mongodb to mockClient
```

Comment:
@rkamysz have you figured out a way we could swap out the mongodb registered in fastify with a custom one?

Reply:
Can we close this PR?